src/pages/Detail/ProductDetail/ProductDetail.js

```diff
@@ -0,0 +1,157 @@
+    };
+  }
+
+  changeImg = idx => {
+    this.setState({ select: idx });
+    console.log(idx);
+  };
+
+  showDesc = () => {
+    this.setState({ showDesc: !this.state.showDesc });
+  };
+
+  showMore = () => {
+    this.setState({ showMoreClicked: !this.state.showMoreClicked });
+  };
+
+  render() {
+    console.log(this.props);
+    const {
+      categoryList,
+      imgList,
+      product_name,
+      summary,
+      description,
+      optionList,
+      nutritionList,
+    } = this.props;
+
+    const { select, showMoreClicked, showDesc } = this.state;
+
+    return (
+      <div className="detail">
+        <nav className="detailNav">
+          <span>
+            Products /{' '}
+            {categoryList.map(category => {
+              return (
+                <a
+                  href={'/products?category=' + category.category_id}
+                  key={category.category_id}
+                >
+                  <span>{category.category_name} </span>
+                </a>
+              );
+            })}{' '}
+            / {product_name}
+          </span>
+        </nav>
+        <div className="productMain">
+          <div className="productImgSection">
+            <ul className="gallery">
+              {imgList.map((img, idx) => {
+                return (
+                  <li key={idx} onClick={() => this.changeImg(idx)}>
+                    <div className="galleryImgBox">
+                      <img
+                        src={img}
+                        alt="cheese"
+                        className={
+                          idx === select ? 'galleryImg border' : 'galleryImg'
+                        }
+                      />
+                    </div>
+                  </li>
+                );
+              })}
+            </ul>
+            <div className="productImgContainer">
+              {imgList.length > 0 && (
+                <img
+                  src={imgList[select]}
+                  alt="selected"
+                  className="selectedImg"
+                />
+              )}
+            </div>
+          </div>
+          <div className="productDescSection">
+            <div className="productDescWrapper">
+              <div className="productMainPrice">
+                {optionList.map(option => {
+                  return (
+                    <span key={option.id} className="price">
+                      <span>${option.price}</span>
+                      <span> | {option.weight}g </span>
+                    </span>
+                  );
+                })}
+              </div>
+              <div className="productTitle">
+                <h1>{product_name}</h1>
+              </div>
+              <div className="productDescWrapper">
+                <div className="summary">{summary}</div>
+                <div
+                  className={
+                    showMoreClicked ? 'productDesc' : 'productDesc hide'
+                  }
```

Comment:
```suggestion
                  className={`productDesc ${showMoreClicked ? '' : 'hide'}`}
```
템플릿 리터럴을 사용하면 이렇게도 가능합니다!

Reply:
오오 꿀팁 감사합니다! 유용해요